### PR TITLE
Demo, and docs use object instead of hash

### PR DESCRIPTION
**Pull request by EWhite613**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG
* Demo, and docs use object instead of hash (For Emb

### CODE REVIEW

File: tests/dummy/app/pods/demo/error/template.hbs [hunk 1]
```diff
@@ -44,10 +44,10 @@
     isVisible=false
   )
   hook='error-dialog'
-  footer=(component 'frost-loading' class='frost-footer-component')
+  footer=(component 'frost-loading' hook='frost-loading' class='frost-footer-component')
```
Comment (notmessenger) [inline tests/dummy/app/pods/demo/error/template.hbs:48]: What is the need for this change?  There are no corresponding changes to any tests.

**Review by notmessenger — CHANGES_REQUESTED** [review]
This usage does not need changed either?

https://github.com/ciena-frost/ember-frost-modal/blob/57fcedcb810a645fa3db04fa7bdc45d0aefe7d0b/addon/templates/components/frost-modal-binding.hbs#L4

If the only change required was to the usage of this component and not to the component itself then this required usage needs to be documented so consumers are aware of its need.

**Comment by notmessenger** [comment]
To solve the failing build you must add a **devDependency** entry of `"bower": "^1.8.2"`

File: tests/dummy/app/pods/demo/error/template.hbs (hunk 1 above)
Comment (EWhite613) [reply tests/dummy/app/pods/demo/error/template.hbs:48]: demo page breaks because of missing `hook` property

Comment (notmessenger) [reply tests/dummy/app/pods/demo/error/template.hbs:48]: 👍

**Review by notmessenger — APPROVED** [review]

**Comment by notmessenger** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-modal/pull-request/124/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-modal)

**Pull request merged**